src/components/Model3d/index.tsx

```diff
@@ -7,7 +8,20 @@ const Model = (props: {
   className?: string
   style?: React.CSSProperties
   onClick?: React.EventHandler<React.SyntheticEvent>
+  onLoad?: () => void
 }) => {
+  const ref = useRef<any>()
+  useEffect(() => {
+    if (props.onLoad && ref.current) {
```

Comment:
`ref.current` should be cached in the hook since it may change when `return callback` is called.